.eslintrc

```diff
@@ -187,7 +187,7 @@
     "space-return-throw-case": [2],
     "space-unary-ops": [2, {"words": true, "nonwords": false}],
     "spaced-line-comment": [0, "always"],
-    "strict": [2, "never"],
+    "strict": 0,
```

Comment:
instead of chaning this, we could disable this rule with inline comment on the specific files